source/_components/climate.intesishome.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: "IntesisHome AC Controller"
+description: "Instructions how to integrate IntesisHome AC devices with Home Assistant"
+date: 2019-03-12 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: intesishome.png
```

Comment:
I cannot find this file, did you forget to add it?

Reply:
Sorry missed that step! Added now, cheers